packages/mjml-head-font/README.md

```diff
@@ -25,9 +28,11 @@ This tag allows you to import fonts if used in your MJML document
   </a>
 </p>
 
+<aside class="notice">
+  If you don't get a CSS file, model one on the "href" file in the example.
+</aside>
 
-attribute            | unit          | description                    | default value
----------------------|---------------|--------------------------------|---------------
-href                 | string        | url of the font                | n/a
-name                 | string        | name of the font               | n/a
-
+attribute   | unit     | description                             | default value
+------------|----------|-----------------------------------------|---------------
+href        | string   | URL of a hosted CSS file for the font   | n/a
```

Comment:
`URL of a CSS file` no need `for the font`

Reply:
corrected